addons/LPCAnimatedSprite/LPCSpriteSheet.gd

```diff
@@ -12,10 +12,11 @@ enum SpriteTypeEnum {
 	OversizeRod,
 	OversizeThrust,
 	OversizeSlash,
+	OversizeSlashReverse,
```

Comment:
Might I suggest some (##) comments to help explain new users what these options mean when selecting their value in the enum?